easybuild/easyconfigs/a/Automake/Automake-1.16.1-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,41 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Uni.Lu/LCSB, NTUA
+# Authors::   Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'Automake'
+version = '1.16.1'
+
+homepage = 'https://www.gnu.org/software/automake/automake.html'
+
+description = "Automake: GNU Standards-compliant Makefile generator"
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['608a97523f97db32f1f5d5615c98ca69326ced2054c9f82e65bade7fc4c9dea8']
+
+builddependencies = [
+    ('binutils', '2.34'),
```

Comment:
this also depends on a recent perl see https://github.com/easybuilders/easybuild-easyconfigs/issues/1822

Reply:
fixed in 7a16d59d7c